src/databricks/sqlalchemy/test_local/test_types.py

```diff
@@ -111,7 +111,7 @@ def test_numeric_renders_as_decimal_with_precision(self):
         )
 
     def test_numeric_renders_as_decimal_with_precision_and_scale(self):
-        return self._assert_compiled_value_explicit(
+        self._assert_compiled_value_explicit(
```

Comment:
I found during testing that this test case would always pass because prefixing an assertion with `return` always evaluates as a pass. When I removed the `return`, the test failed for TINYINT types (should have been caught during initial development). This test now passes.